forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java

```diff
@@ -62,8 +62,8 @@ private static Hash commitSquashed(PullRequest pr, Repository localRepo, Hash fi
     private static Hash fetchMergeSource(PullRequest pr, Repository localRepo) throws IOException, CommitFailure {
         var sourceMatcher = mergeSourcePattern.matcher(pr.title());
         if (!sourceMatcher.matches()) {
-            throw new CommitFailure("Could not determine the source for this merge. A Merge PR title must be specified on the format: " +
-                                            "Merge `project`:`branch` to allow verification of the merge contents.");
+            throw new CommitFailure("Could not determine the source for this merge. A Merge PR title must be specified of the format: `" +
```

Comment:
Perhaps "specified *in* the format:" would sound better, but I'm not a native speaker. :)

Reply:
Yeah, that sounds nicer (though I think technically both are correct).